drone exec panic when missing interpolation variables

example yaml file:

```
pipeline:
  build:
    image: plugins/docker
    repo: ${DRONE_REPO}
    tag: ${DRONE_COMMIT}
```

panic output:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0xa0 pc=0x305368]

goroutine 1 [running]:
panic(0x7f8cc0, 0x820ed20c0)
    /usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/drone/drone/yaml/transform.argsToEnv(0x8210d6b40, 0x8210d6c60, 0x0, 0x0)
    /go/src/github.com/drone/drone/yaml/transform/util.go:23 +0x258
github.com/drone/drone/yaml/transform.PluginParams(0x8210e0580, 0x0, 0x0)
    /go/src/github.com/drone/drone/yaml/transform/plugin.go:41 +0x12e
github.com/drone/drone/agent.(*Agent).prep(0x8213d5180, 0x8210e0180, 0xd9aec0, 0x0, 0x0)
    /go/src/github.com/drone/drone/agent/agent.go:163 +0xcd5
github.com/drone/drone/agent.(*Agent).Run(0x8213d5180, 0x8210e0180, 0x8210ac8c0, 0x0, 0x0)
    /go/src/github.com/drone/drone/agent/agent.go:56 +0xa9
main.exec(0x8210abd40, 0x0, 0x0)
    /go/src/github.com/drone/drone/drone/exec.go:420 +0x21e6
main.glob.func10(0x8210abd40)
    /go/src/github.com/drone/drone/drone/exec.go:27 +0x25
github.com/drone/drone/vendor/github.com/codegangsta/cli.Command.Run(0x8f0be0, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x97c790, 0x15, 0x0, ...)
    /go/src/github.com/drone/drone/vendor/github.com/codegangsta/cli/command.go:131 +0x1058
github.com/drone/drone/vendor/github.com/codegangsta/cli.(*App).Run(0x8210abb00, 0x820f4a060, 0x2, 0x2, 0x0, 0x0)
    /go/src/github.com/drone/drone/vendor/github.com/codegangsta/cli/app.go:175 +0xfd1
main.main()
```

--- a/yaml/transform/util.go
+++ b/yaml/transform/util.go
@@ -15,6 +15,11 @@ import (
 func argsToEnv(from map[string]interface{}, to map[string]string) error {
 
 	for k, v := range from {
+		if v == nil {
+			to[k] = ""
+			continue
+		}
+
 		t := reflect.TypeOf(v)
 		vv := reflect.ValueOf(v)
 

